src/components/content/news/news.php

```diff
@@ -2,12 +2,18 @@
 <div class="flex justify-center mt-10">
   <section class="w-4/5 mt-2 flex flex-col gap-4">
     <?php for ($i = 0; $i <= 10; $i++) { ?>
-      <article class="flex flex-col gap-2">
-        <span class="text-[10px] text-white bg-sky-600 p-1 w-fit">2022/10/10</span>
-        <h3 class="text-base">ゴルフシーズン期間限定キャンペーン実施中</h3>
-        <p class="text-[10px] text-gray-600">コンセプトは、「楽しく」「もっと早く上達を！」です。東京から60kmと大変近く、人気の基礎ゴルフ場GOLF STUDIOで開催しています。ゴルフ界隈ではタブーな打ち直しが...</p>
-        <hr class=" border border-solid border-gray-300" />
-      </article>
+      <a href="#">
+        <article class="flex gap-2 border-b pb-5">
+          <span class="text-[8px] text-white bg-sky-600 px-2 w-fit h-4">
+            2022/10/10
+          </span>
+          <p class="text-[16px] text-gray-600 font-normal leading-4">
```

Comment:
[quesiton]
これとほとんど同じかなと思ったけど違う？👀

```suggestion
          <p class="text-base text-gray-600">
```

Reply:
ほとんど同じで大丈夫かと思いあまり訂正しませんでした、、少しだけ修正加えました。一から作った方がよかったでしょうか？